src/test_utils/kbn_server.ts

```diff
@@ -75,7 +75,7 @@ export function createRootWithSettings(
       repl: false,
       basePath: false,
       optimize: false,
-      oss: false,
+      oss: true,
```

Comment:
__note:__ NP x-pack plugins are picked up by Kibana if `oss` is `true` and that's not what we want by default when we run test servers in OSS.